src/cf.data.pre

```diff
@@ -7112,11 +7112,22 @@ LOC: Config.visibleHostname
 DEFAULT: none
 DEFAULT_DOC: Automatically detect the system host name
 DOC_START
-	If you want to present a special hostname in error messages, etc,
-	define this.  Otherwise, the return value of gethostname()
-	will be used. If you have multiple caches in a cluster and
-	get errors about IP-forwarding you must set them to have individual
-	names with this setting.
+	The public domain name (FQDN) of this Proxy.
+
+	This name will be used in generated URLs sent to clients for
+	objects they need to download directly from Squid, and for other
```

Comment:
It would be good to add an example to the first part of the sentence. For example: `objects they need to download directly from Squid (e.g., style sheets for error pages and FTP icons)`.

Reply:
resolved by the other change to this sentence